includes/core/forms/default-form.php

```diff
@@ -101,19 +134,20 @@ public function print_custom_fields() {
 		$html = '';
 
 		if ( ! empty( $this->custom_fields ) && is_array( $this->custom_fields ) ) {
-			foreach ( $this->custom_fields as $k => $v ) {
 
-				switch ( $v['type'] ) {
-					case 'payment_button':
-						$html .= Fields\Payment_Button::html( $v );
-						break;
-					case has_filter( 'simpay_custom_fields' ):
-						$html .= apply_filters( 'simpay_custom_fields', $html, $v );
-						break;
-				}
```

Comment:
This broke the payment button output.

Reply:
Unsure here.

See https://github.com/wpsimplepay/WP-Simple-Pay-Pro-3/commits/master/includes/core/forms/default-form.php

Also https://github.com/wpsimplepay/WP-Simple-Pay-Pro-3/issues/576